WtWSMS/common/casus_belli_types/BP_roman_reconquest.txt

```diff
@@ -286,5 +285,273 @@ the_roman_empires_imperial_reconquest_cb = {
 	max_ai_diplo_distance_to_title = 500
 	
 	on_invalidated_desc = msg_invalidate_war_title
+}
+
+#Cribbed from Vanilla CK3 Imperial Reconquest
+successor_imperial_reconquest_cb = {
+	group = de_jure
+
+	#you should at least have to siege down something, and the wars should have the potential to be longer and costlier
+	max_attacker_score_from_battles = 75
+
+	combine_into_one = yes
+	should_show_war_goal_subview = yes
+	mutually_exclusive_titles = { always = yes }
+
+	allowed_for_character = {
+		scope:attacker = {
+			NOR = {
+				has_title = title:e_roman_empire
+				has_title = title:e_wre
+				has_title = title:e_byzantium
+			}
+			valid_reconquest_culture_trigger = { CULTURE = scope:attacker.culture }
+		}
+	}
+	
+	allowed_against_character = {
+		scope:defender = {
+			NOT = { faith.religion = religion:islam_religion }
+			NOR = {
+				has_title = title:e_wre
+				has_title = title:e_byzantium
+				has_title = title:e_roman_empire
+			}
+			NOR = {
+				scope:defender = { target_is_liege_or_above = scope:attacker }
+				scope:attacker = { target_is_liege_or_above = scope:defender }
+			}
+		}
+	}
+
+	target_titles = neighbor_land_or_water
+	target_title_tier = tier_kingdom
+	target_de_jure_regions_above = yes
+	ignore_effect = change_title_holder
+	ai_only_against_neighbors = yes
+	
+	cost = {
+		gold = successor_reconquest_war_gold_cost
+		prestige = successor_reconquest_war_fame_cost
+	}
+	
+	#Changed from CK2: Requires the associated title to be dead
```

Comment:
The problem with this one would be if you have conflicting legitimacies, with several pretenders.

Reply:
Hanging comment. Fixed.